lib/iris/tests/results/imagerepo.json

```diff
@@ -6,7 +6,8 @@
     "gallery_tests.test_plot_COP_maps.TestCOPMaps.test_plot_cop_maps.0": [
         "https://scitools.github.io/test-iris-imagehash/images/v4/ea9138db95668524913e6ac168997e85957e917e876396b96a81b5ce3c496935.png",
         "https://scitools.github.io/test-iris-imagehash/images/v4/ea9130db95668524913c6ac178995b0d956e917ec76396b96a853dcf94696935.png",
-        "https://scitools.github.io/test-iris-imagehash/images/v4/ea9130db95668524913e6ac168991f0d956e917ec76396b96a853dcf94796931.png"
+        "https://scitools.github.io/test-iris-imagehash/images/v4/ea9130db95668524913e6ac168991f0d956e917ec76396b96a853dcf94796931.png",
```

Comment:
Changes in this file are expounded on in this comment: https://github.com/SciTools/iris/pull/3762#issuecomment-680249000. I've been over these changes with @trexfeathers and they all seem appropriate.